CoroutineTemplate/buildSrc/src/main/java/Versions.kt

```diff
@@ -1,5 +1,5 @@
 object Versions {
-    const val BUILD_GRADLE_VERSION = "4.2.1"
+    const val BUILD_GRADLE_VERSION = "7.0.2"
```

Comment:
Shouldn't this be `7.1.2` or higher? 🤔

Reply:
Good catch, @ryan-conway 👍  I follow my `Android studio suggest` actually but yeah, Let's me try to upgrade to `7.1.2` 🙏  If everything works then we can go with this version 🚀